app/models/name/parse.rb

```diff
@@ -122,7 +122,7 @@ def match?(str)
     #{ANY_AUTHOR_ABBR} |
     van\s | d[eu]\s |
     [A-ZÀÁÂÃÄÅÆÇĐÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞČŚŠ(] |
```

Comment:
Line 124. Can/Should the list of letters be replaced by `/\p{Latin}/u`

Reply:
Perhaps, but I consider that outside the scope of this PR.